recipes/guietta/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "guietta" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6154aa2f763058df5bdbd10a9d3baf0911a1ad803ce7c7b9383baccef5c081bb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
Can delete `build` section and just add `setuptools` to `host`.

Reply:
Done!